source/assets/components/recipeCard/recipeCard.js

```diff
@@ -48,6 +48,13 @@ class recipeCard extends HTMLElement {
     // this will reset the struction of the section element within our shadow root
     this.shadowRoot.querySelector("section").innerHTML = `
       <div class="row">
+        <div class="col-xs-1">
+          <div class="favorite-button">
+            <button>
```

Comment:
What do you think about moving the `id="fav-btn"` from `img` to `<button>`? 

i.e. `<button id="fav-btn">`

Reply:
That seems fine to do. Shouldn't break anything

Reply:
Thanks! It will help with testing